tests/foreman/cli/test_satellitesync.py

```diff
@@ -1061,28 +1060,54 @@ def test_positive_export_import_redhat_cv_with_huge_contents(
 
         :CaseLevel: Acceptance
         """
-        rhel_repo_name = 'rhscl7'
-        product = 'rhscl'
-        releasever = '7Server'
-        rhel_repo = _enable_rhel_content(
-            default_sat, class_export_entities['exporting_org'], rhel_repo_name, releasever, product
+        cv_name = gen_string('alpha')
+
+        with manifests.clone() as manifest:
+            default_sat.put(manifest, manifest.filename)
+        Subscription.upload({'file': manifest.filename, 'organization-id': function_org.id})
+        RepositorySet.enable(
+            {
+                'basearch': 'x86_64',
+                'name': REPOSET['rhscl7'],
```

Comment:
Is there some specific reason to use rhscl or is it just a random RH repo?

Reply:
Some reasonably large repo needs to be used. RHSCL was chosen by original author @latran. This is only a fix of failed test.